tools/eslint-config-web3-base/ts.js

```diff
@@ -0,0 +1,86 @@
+module.exports = {
+	parser: '@typescript-eslint/parser',
+	plugins: ['@typescript-eslint'],
+	extends: [
+		'airbnb-base',
+		'eslint:recommended',
+		'plugin:@typescript-eslint/recommended',
+		'plugin:@typescript-eslint/recommended-requiring-type-checking',
+		'prettier',
+		'plugin:import/errors',
+		'plugin:import/warnings',
+		'plugin:import/typescript',
+	],
+	rules: {
+		'class-methods-use-this': ['error'],
+		'no-unused-expressions': ['error'],
+		'no-continue': 'off',
+		'no-underscore-dangle': ['error'],
+		'import/prefer-default-export': 'off',
```

Comment:
I think this is a good rule to keep enabled

Reply:
I don't prefer enforcing default export in all the files, if its just one export. Named exports are much readable and stay consistent in the code.